scipy/stats/_distribution_infrastructure.py

```diff
@@ -3510,10 +3510,17 @@ def make_distribution(dist):
             is defined. The preferred interface may change in future SciPy versions,
             in which case support for an old interface version may be deprecated
             and eventually removed.
-        parameters : dict
-            Each key is the name of a parameter,
+        parameters : dict or tuple
+            If a ``tuple`` it should be a ``tuple`` of ``dict``s with the structure
+            described below. Each ``dict`` in the ``tuple`` will correspond to a
+            separate parametrization of the distribution. If multiple parameterizations
+            are specified, ``dist``'s class must also define a method
+           ``_process_parameters`` to allow mapping between the different parameterizations.
```

Comment:
Right now, this looks redundant. If you intended it to be, maybe mention "see below".
```suggestion
            separate parametrization of the distribution. If multiple parameterizations
            are specified, ``dist``'s class must also define a method
           ``_process_parameters`` to map between the different parameterizations.
```

But actually, I think this will be the exception rather than the rule. I would start by describing only the dict. So basically, undo this change at the top, and modify only the first line: "If a dict, each key is the name of a parameter,...."

Then afterward you can describe what a *tuple* should be. I think this would both eliminate redundancy and be easier to read, tbh.

Reply:
Sounds good.